en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/configurations/configuration-overview.md

```diff
@@ -2,16 +2,76 @@
+    | Log Configuration File  | `logconfig-toml-configmap.yaml` |
+    | Log4j2 Configuration File      | `log4j2-configmap.yaml` |
+
+    | **Mode** | **Directory (File path)** |
+    |----------|---------------------------|
+    | [Choreo Connect as a Standalone Gateway]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/concepts/as-a-standalone-gateway/) | `<CHOREO-CONNECT_HOME>/k8s-artifacts/choreo-connect` |
+    | [Choreo Connect with WSO2 API Manager as a Control Plane]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/concepts/apim-as-control-plane/) | `<CHOREO-CONNECT_HOME>/k8s-artifacts/choreo-connect-with-apim/choreo-connect` |
+
+??? info "**For Kubernetes with Helm Charts**"
+
+    You may update the configurations in the following ways, when deploying with Helm Charts and the resources from the [Choreo Connect Helm Chart Git repository](https://github.com/wso2/kubernetes-microgateway/tree/1.1.x/helm/choreo-connect). 
+
+    - By updating [values.yaml](https://github.com/wso2/kubernetes-microgateway/blob/1.1.x/helm/choreo-connect/values.yaml). [Click here for the steps.](https://apim.docs.wso2.com/en/latest/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/deploy/cc-on-kubernetes-with-apim-as-control-plane-helm-artifacts/#step-2-update-the-deployment-configurations)
+    - By using --set with the Helm command. [Click here for an example.](https://apim.docs.wso2.com/en/latest/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/deploy/cc-on-kubernetes-with-apim-as-control-plane-helm-artifacts/#step-2-install-chart)
+
+    You can find the [complete list of parameters here](https://github.com/wso2/kubernetes-microgateway/blob/master/helm/choreo-connect/README.md#configuration).
+
+    These configuration parameters are based on the [templates available in here](https://github.com/wso2/kubernetes-microgateway/tree/1.1.x/helm/choreo-connect/templates). The `values.yaml` file and the templates are common to both of the modes, [Choreo Connect as a Standalone Gateway]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/concepts/as-a-standalone-gateway/) and [Choreo Connect with WSO2 API Manager as a Control Plane]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/concepts/apim-as-control-plane/). The mode itself can be updated as given below.
+
+    ```
+    wso2:
+      deployment:
+        mode: "APIM_AS_CP"
+    ```
+
+    ```
+    wso2:
+      deployment:
+        mode: "STANDALONE"
+    ```
+
+    You may find the template file for the main Choreo Connect configuration and logging in the following names.
+
+    | **File** | **File Name** |
+    |----------|---------------|
+    | Configuration File   | `config-toml-configmap.yaml` |
+    | Log Configuration File  | `logconfig-toml-configmap.yaml` |
+    | Log4j2 Configuration File  | `enforcer-log4j2-configmap.yaml` |
 
```

Comment:
We can add the following as well.

If you have advanced configurations that are not templated, you can have your own **templated** configuration file and set it in the Helm Chart as follows. Make sure to have existing templates already provided by the Helm Chart within your configuration files, otherwise you may lose configurations set with the values.yaml file.

```
--set-file wso2.deployment.adapter.configToml=<File Path For Templated Config Toml >
--set-file wso2.deployment.adapter.logConfigToml=<File Path For Templated Log config Toml >
```